backend/inspirehep/tools/utils.py

```diff
@@ -77,7 +77,10 @@ def find_references(references, requested_format):
         elif len(hits) > 1:
             errors.append({"ref": ref, "line": line, "type": "ambiguous"})
         else:
-            ret.append(hits[0]["_source"][display_format])
+            source_field = hits[0]["_source"]
+            control_number = source_field["control_number"]
+            texkey = getattr(source_field, "texkeys", [control_number])[0]
```

Comment:
just curious: what's wrong with
```suggestion
            texkey = source_field.get("texkeys", [control_number])[0]
```
?